Documentation-Requirements.md

```diff
@@ -100,6 +100,15 @@ utility.
 
   ```
 
+- Long lines should not be folded across multiple lines using the '`\`' continuation character.
```

Comment:
Suggested rewrite of lines 103 - 110: 
Long lines should not span across multiple lines by using the '`\`' continuation character.

GitHub automatically renders such blocks with scrollbars. Consequently, backslash continuation characters are not necessary and are a visual distraction. These characters also mess up a user's shell history when commands are pasted into a terminal.

Reply:
Thanks @klynnrif - branch updated.